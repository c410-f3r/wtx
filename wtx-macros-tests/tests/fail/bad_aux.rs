#[wtx_macros::pkg(data_format(json), id(FooId))]
mod pkg {
  #[pkg::aux]
  impl Foo {
    type Foo = i32;
  }

  #[pkg::req_data]
  struct Req(
    i32
  );

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
