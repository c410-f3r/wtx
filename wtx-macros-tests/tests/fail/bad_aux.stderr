error: #[pkg::aux] must be an item implementation with none, one `#[pkg::aux_data]`, one `#[pkg::aux_params]` or both `#[pkg::aux_data]` and `#[pkg::aux_params]`
 --> tests/fail/bad_aux.rs:3:10
  |
3 |   #[pkg::aux]
  |          ^^^

error: aborting due to 1 previous error

